app/services/project_data_service.rb

```diff
@@ -121,7 +121,7 @@ def report_runtime_version(dependency)
         "runtime_version:#{dependency.version == 'unknown version' ? 'none' : dependency.version.delete('^0-9.')}",
         "project:#{@project.name}",
         "criticality:#{@project.criticality}",
-        "tags:#{@project.tags&.none? ? 'none' : @project.tags.join(':')}"
+        "tags:#{@project.tags&.nil? ? 'none' : @project.tags.join(':')}"
```

Comment:
You could use `.blank?` here if you want to send `none` for `nil` or `[]` values.

Reply:
Good call. Have added in [dd57686](https://github.com/artsy/horizon/pull/586/commits/dd576864fcee919d6e55174f94203f973efc4d5f)